capabilities_matrix/_topics/infrastructure_providers.md

```diff
@@ -2,67 +2,67 @@
 
  The following tables outline the capabilities of {{ site.data.product.title_short }} that are known to work for various virtual infrastructure providers.
 
-| Discovery                                                    | vSphere | oVirt / RHV | OpenStack undercloud    | IBM Power HMC | KubeVirt / OSV |
-| ------------------------------------------------------------ | ------- | ----------- | ----------------------- | ------------- | -------- |
-| Compute Inventory                                            | ✅      | ✅          | ✅ (Nodes and Services) | ✅            | ✅         |
-| Network Inventory                                            | ✅      | ✅ (OVN)    | ❌                      | ✅            | ❌          |
-| Storage Inventory                                            | ✅      | ❌          | ❌                      | ✅            | ✅         |
-| Events                                                       | ✅      | ✅          | ✅                      | ✅            | ❌          |
-| Metrics                                                      | ✅      | ✅          | ✅                      | ✅            | ❌          |
-| Forensic Analysis (SmartState)                               | ✅      | ✅          | ✅ (Nodes)              | ❌            | ❌          |
+| Discovery                                                    | vSphere | oVirt / RHV | OpenStack undercloud    | IBM Power HMC | KubeVirt | OSV |
```

Comment:
So we show the other subclassed providers together, `oVirt / RHV`
Could we do `| KubeVirt / OSV |` at least until we have any differentiation between the two in features supported

Reply:
I think we still need to update the capability matrix with KubeVirt / OSV

Reply:
That is there https://github.com/ManageIQ/manageiq-documentation/blob/master/capabilities_matrix/_topics/infrastructure_providers.md, the earlier change was just to separate it but no longer doing that

Reply:
:facepalm: I didn't see that in the diff, thanks